### DISCUSSION TITLE
useFieldArray

### DISCUSSION THREAD

**Discussion by Joan-George** [react-hook-form/react-hook-form#9431] [Q&A] [answered]
Can I use more than use useFieldArray in a form?

`const { fields, append, remove} = useFieldArray({
    control, 
    name: "fieldArray1",
  })`
`const { fields, append, remove} = useFieldArray({
    control, 
    name: "fieldArray2",
  })
`

Is this possible to use the useFieldArray hook like the example mentioned above?
If not possible how can use two useFieldArray hooks in a form?

**Comment by bluebill1049** [comment, marked as answer]
Yes, that's correct. you can do that.

**Reply by Joan-George** [reply]
Can I create more than one useFieldArray dynamically with an array of objects?

If It is possible can you give me an example pls?
